packages/server/codegen.yml

```
@@ -1,5 +1,5 @@
 overwrite: true
-schema: src/**/*.graphql
+schema: ./src/**/*.graphql
```

```suggestion
schema: src/**/*.graphql
```
this is actually not needed.